lingua_franca/lang/parse_en.py

```diff
@@ -671,15 +672,15 @@ def extract_duration_en(text, replace_with=""):
         text = re.sub(unit_pattern, replace_with, text)
 
         if unit == "months":
-            time_units["days"] += 30 * value
+            time_units["days"] += DAYS_IN_1_MONTH * value
```

Comment:
This isn't a consistent figure. I think the best route is to rely on the datetime module, increment the `month` attribute, and distinguish between '30 days' and 'one month'.

Reply:
the datetime module provides timedelta objects, but the highest resolution they provide is in days. This makes sense since we can define anything from days and bellow in terms of SI units (seconds). 

Since this represents a time interval (a duration) it must mean the same in whatever reference date. if a month has a different duration depending on reference date it can not be used for measurement of an interval

Silly example, imagine watching a video that randomly lasted 28 or 31 days....

Reply:
also note, this belong in pr#95 which is required for this PR, please move discussion there